test/GPUReg.jl

```diff
@@ -83,19 +83,6 @@ end
     @test size(res) == (32,)
     @test res2 == res
 
-    # measure! and reset
```

Comment:
Why tests is rm?

Reply:
Because these tests are ill defined. They should not work. Note the previous measurement on operators are not correct.